From 5a93d9c2db0bcd07741c4e20aff128a91e95b28a Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Mon, 9 Oct 2017 08:27:59 +0200
Subject: [PATCH 11/14] topic_miamimp: Quietly boot kernel and skip bootdelay

Using "earlycon" is nice in early development, but now things are getting
stable, we much prefer a quiet but faster boot.

Pass "quiet" to kernel to set loglevel to warnings only.
Set BOOTDELAY to "0" to skip waiting for user input.
---
 configs/topic_miamimp_defconfig | 1 +
 include/configs/topic_miamimp.h | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/configs/topic_miamimp_defconfig b/configs/topic_miamimp_defconfig
index 940a47e..3b7112e 100644
--- a/configs/topic_miamimp_defconfig
+++ b/configs/topic_miamimp_defconfig
@@ -18,6 +18,7 @@ CONFIG_FIT_VERBOSE=y
 CONFIG_SPL_LOAD_FIT=y
 # CONFIG_DISPLAY_CPUINFO is not set
 # CONFIG_DISPLAY_BOARDINFO is not set
+CONFIG_BOOTDELAY=0
 CONFIG_SPL=y
 CONFIG_SPL_SYS_MALLOC_SIMPLE=y
 CONFIG_SPL_OS_BOOT=y
diff --git a/include/configs/topic_miamimp.h b/include/configs/topic_miamimp.h
index 67bfada..3e56df4 100644
--- a/include/configs/topic_miamimp.h
+++ b/include/configs/topic_miamimp.h
@@ -45,7 +45,7 @@
 	"kernel_size=0x1e00000\0" \
 	"fdt_size=0x80000\0" \
 	"bootenv=uEnv.txt\0" \
-	"bootargs=earlycon clk_ignore_unused\0" \
+	"bootargs=quiet clk_ignore_unused\0" \
 	"loadbootenv=load mmc $sdbootdev:$partid ${loadbootenv_addr} ${bootenv}\0" \
 	"importbootenv=echo Importing environment from SD ...; " \
 		"env import -t ${loadbootenv_addr} $filesize\0" \
-- 
1.9.1

